### DISCUSSION TITLE
Cant Connect beats agent to Elasticsearch and Kibaba on port 8220 and 9200 on Securityonion

### DISCUSSION THREAD

**Discussion by seyiakilude** [Security-Onion-Solutions/securityonion#11753] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

16GB

### Storage for /

250GB

### Storage for /nsm

1,015.75 GB / 558.69 GB Free

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I am trying to import Netflow data from a windows machine with filebeats installed but cant connect to Securityonion on port 8220 for Kibana and 9200 on Elasticsearch.

The error message is no response on the ports respectively.

![WhatsApp Image 2023-11-08 at 23 15 38](https://github.com/Security-Onion-Solutions/securityonion/assets/25613756/46fc33da-ebd3-4e9f-a216-c3be070c2e4a)

![WhatsApp Image 2023-11-08 at 23 17 37](https://github.com/Security-Onion-Solutions/securityonion/assets/25613756/b4ec80f1-0168-45cb-b26d-acc06a0e12bf)

![WhatsApp Image 2023-11-08 at 23 16 49](https://github.com/Security-Onion-Solutions/securityonion/assets/25613756/49321062-a5c0-484e-b138-5ff2bb68140b)

![WhatsApp Image 2023-11-08 at 23 16 02](https://github.com/Security-Onion-Solutions/securityonion/assets/25613756/17c6d488-cd9d-4e48-8dda-3c77bb7ff956)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by seyiakilude** [comment]
Here is me trying again
![Uploading Image 2023-11-10 at 17.19.jpeg…]()

**Comment by dougburks** [comment]
You say above that you are running version 2.4.10. However, your screenshots appear to show an older version.

Please perform a fresh installation of 2.4.20 or the latest version shown at https://securityonion.net/download. You can then deploy the Elastic Agent to endpoints as shown at https://docs.securityonion.net/en/2.4/elastic-agent.html.
